lib/SILOptimizer/Utils/Generics.cpp

```diff
@@ -1969,6 +1997,10 @@ static ApplySite replaceWithSpecializedCallee(ApplySite applySite,
     assert(resultBlock->getSinglePredecessorBlock() == tai->getParent());
     auto *newTAI = builder.createTryApply(loc, callee, subs, arguments,
                                           resultBlock, tai->getErrorBB());
+    FullApplySite(newTAI).insertAfterFullEvaluation(
+        [&](SILBasicBlock::iterator insertPt) {
+          cleanupCallArguments(builder, loc, arguments, argsNeedEndBorrow);
```

Comment:
Hmmm... I think I need to create a new SILBuilder here.

Reply:
I am going to try to break this. This to not fail says we are missing code coverage.